### PR TITLE
inconsistent allignment in list of child process for gropher proc command

### LINKED ISSUE #45: [BUG] allignment in list of child processes in `grofer proc -p PID` is not consistent.
**Describe the bug**
The alignment in the table for the list of child processes of a process is not consistent. I'm assuming this is happening due to the varying lengths of PID. 

**To Reproduce**
Run `grofer proc -p 1` and hit the `↓` key a few times.

**Expected behavior**
Alignment remains consistent irrespective of the length of the PID. 

**Screenshots**
Screenshot_20200819_231213

**Desktop (please complete the following information):**
 - OS: Kubuntu 20.04
 - Go Version: 1.14.7
- Terminal: Konsole

**Additional context**
Should be fixable by looking at the code in the `getChildProcs` function in the file `src/display/process/procGraphs.go`

### PR DESCRIPTION
**Pull request by siddhant94**
# Description

1) Inconsistent allignment in list of child process for command `gropher proc -p 1`.
2) Follow list color-scheme as in AllProcPage

Fixes #45 

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

# Checklist:

- [x] I have read the [contribution guidelines](https://github.com/pesos/grofer/blob/master/CONTRIBUTING.md) and followed it as far as possible. 
- [x] I have performed a self-review of my own code (if applicable)
- [x] I have commented my code, particularly in hard-to-understand areas (if applicable)
- [x] I have run `go fmt` on my code ([reference](https://blog.golang.org/gofmt))
- [x] I have made corresponding changes to the documentation (if applicable)
- [x] My changes generate no new warnings 
- [x] Any dependent and pending changes have been merged and published

### CODE REVIEW

File: src/display/process/procGraphs.go [hunk 1]
```diff
@@ -30,21 +31,21 @@ import (
...
+		spacesForCommandRowData = strings.Repeat(" ", 22-len(processPid))
 		exe, err := proc.Exe()
 		if err == nil {
-			temp := strconv.Itoa(int(proc.Pid))
-			for i := 0; i < 22-len(strconv.Itoa(int(proc.Pid))); i++ {
-				temp = temp + " "
-			}
-			temp = temp + "[" + exe + "](fg:green)"
-			childProcs = append(childProcs, temp)
+			processData = processPid + spacesForCommandRowData + "[" + exe + "](fg:green)"
```
Comment (MadhavJivrajani) [inline src/display/process/procGraphs.go:43]: The PID should be made `yellow` in case there is no error, similar to how it's done in the `else` clause.

**Review by MadhavJivrajani — CHANGES_REQUESTED** [review]

File: src/display/process/procGraphs.go (hunk 1 above)
Comment (siddhant94) [reply src/display/process/procGraphs.go:43]: So in both cases it should be yellow ? Because the implementation I changed had yellow only for else case.

Comment (MadhavJivrajani) [reply src/display/process/procGraphs.go:43]: Yes, it should be yellow uniformly! 😄

Comment (siddhant94) [reply src/display/process/procGraphs.go]: Sure :+1: , updated the PR

File: src/display/process/procGraphs.go [hunk 2]
```diff
@@ -30,21 +31,22 @@ import (
...
-			temp := strconv.Itoa(int(proc.Pid))
-			for i := 0; i < 22-len(strconv.Itoa(int(proc.Pid))); i++ {
-				temp = temp + " "
-			}
-			temp = temp + "[" + exe + "](fg:green)"
-			childProcs = append(childProcs, temp)
+			processData += "[" + exe + "](fg:green)"
 		} else {
-			childProcs = append(childProcs, "["+strconv.Itoa(int(proc.Pid))+"](fg:yellow)"+"            "+"NA")
+			processData += "NA"
```
Comment (Gituser143) [inline src/display/process/procGraphs.go:46]: Hey @siddhant94, Leaving NA white makes this look kind of out of place. Also this colour scheme is kind of out of place from the other table we see when we run `grofer proc`. We're sorry for the inconvenience, but could you get this list to follow the same colour scheme please? 

The colour scheme is something like this:

| Field | Colour |
|--------|----------|
|  pid | white |
| process | green |
| selected row (both pid and process) | yellow |

**Review by Gituser143 — CHANGES_REQUESTED** [review]

**Comment by Gituser143** [comment]
@siddhant94 Apart from the colour scheme change requested, everything else looks great!

File: src/display/process/procGraphs.go (hunk 2 above)
Comment (siddhant94) [reply src/display/process/procGraphs.go:46]: @Gituser143 Sure :+1: 
Thanks for providing the color-scheme. Can you check this screenshot to verify if this is the desired behaviour?
headings -> white, `PID` & `NA` -> white, process -> green & selectedRow -> yellow

bug-fix-screenshot

**Comment by siddhant94** [comment]
> @siddhant94 Apart from the colour scheme change requested, everything else looks great!

Perfect! Let me know if there are any more changes! Have updated the PR with the mentioned color-scheme.

File: src/display/process/procGraphs.go (hunk 2 above)
Comment (Gituser143) [reply src/display/process/procGraphs.go:46]: This looks good! @MadhavJivrajani do you think NA should be green too?

**Review by Gituser143 — APPROVED** [review]

**Comment by Gituser143** [comment]
@siddhant94 Awesome work! I'll wait for @MadhavJivrajani to approve before merging.

**Review by MadhavJivrajani — APPROVED** [review]
Thanks for your contribution @siddhant94! 
Everything looks good to me.

**Pull request merged**